packages/patternfly-4/react-core/src/components/DataList/DataListAction.tsx

```diff
@@ -1,21 +1,42 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/DataList/data-list';
+import { Omit } from '../../helpers/typeUtils';
 
-class DataListAction extends React.Component {
-  constructor(props) {
+export interface DataListActionProps extends Omit<React.HTMLProps<HTMLDivElement>, 'children'> {
+  /** Content rendered as DataList Action  (e.g <Button> or <Dropdown>) */
+  children: React.ReactNode | React.ReactNode[];
```

Comment:
Why do we need `React.ReactNode[]`? All we do with children is render them like `{children}`.